### PR TITLE
Appveyor EDM support

### PR DESCRIPTION
**Pull request by corranwebster**
This PR uses EDM to create test environments for Appveyor CI runs, the same way we are for TravisCI.

### CODE REVIEW

**Comment by corranwebster** [comment]
Looks like there is an issue with finding the `edm` command while running from invoke (at least at first approximation).

File: appveyor.yml [hunk 1]
```diff
@@ -21,13 +23,12 @@ cache:
...
 
 install:
-  - ps: if ((Test-Path "c:/temp") -eq 0) { mkdir c:/temp }
-  - ps: python -m pip install --upgrade --no-binary wheel pip
-  - ps: pip install --upgrade wheel
-  - ps: pip --version
-  - cmd /v:on /e:on /c ".\appveyor-install.cmd"
+  - cmd: appveyor-bootstrap-edm.cmd setup
+  - ps: edm info
+  - cmd /e:on /v:on /c ".\appveyor-install.cmd"
```
Comment (itziakos) [inline appveyor.yml]: there is probably no need for a separate install and test batch files. We can probably call edm run from appveyor.yml directly

File: appveyor.yml [hunk 2]
```diff
@@ -21,13 +23,12 @@ cache:
 init:
   - ps: $Env:sdkbin = "C:\Program Files\Microsoft SDKs\Windows\" + $Env:sdkver + "\Bin"
```
Comment (itziakos) [inline appveyor.yml]: These environment variables are not needed

**Comment by itziakos** [comment]
failures are intermittent #3215 and #3307

**Comment by corranwebster** [comment]
Closed in favour of #298.

**Pull request closed without merging**